presto-main/src/main/java/io/prestosql/operator/scalar/timetz/TimeWithTimeZoneToVarcharCast.java

```diff
@@ -48,18 +48,17 @@ public static Slice cast(@LiteralParameter("p") long precision, @SqlType("time(p
         long nanos = unpackTimeNanos(packedTime);
         int offsetMinutes = unpackOffsetMinutes(packedTime);
 
-        return formatAsString((int) precision, nanos * PICOSECONDS_PER_NANOSECOND, offsetMinutes);
+        return toVarchar((int) precision, nanos * PICOSECONDS_PER_NANOSECOND, offsetMinutes);
```

Comment:
as well as this.
Please make a commit with only `String.format` -> `format` and then other(s) with the rest